platform/mbed_wait_api.h

```diff
@@ -55,9 +55,9 @@ extern "C" {
  *  @param s number of seconds to wait
  *
  *  @note
- *    If the RTOS is present, this function always spins to get the exact number of microseconds,
- *    which potentially affects power (such as preventing deep sleep) and multithread performance.
- *    You can avoid it by using Thread::wait().
+ *    If the RTOS is present, this function spins to get the exact number of microseconds,
+ *    (for 1/100th of sec) which potentially affects power (such as preventing deep sleep) and 
+ *    multithread performance. You can avoid it by using ThisThread::sleep_for() / wait_ms()
```

Comment:
minor nit: This slash made me think math was involved. Maybe say "ThisThread::sleep_for() or wait_ms()" ?